keps/sig-windows/20190103-windows-node-support.md

```diff
@@ -86,20 +87,30 @@ As of 29-11-2018 much of the work for enabling Windows nodes has already been co
 
 ### What works today
 - Windows-based containers can be created by kubelet, [provided the host OS version matches the container base image](https://docs.microsoft.com/en-us/virtualization/windowscontainers/deploy-containers/version-compatibility)
-    - Pod (single or multiple containers per Pod with process isolation)
+    - Pod (single or multiple containers per Pod with process isolation). There are no notable differences in Pod status fields between Linux and Windows containers
     - Services types NodePort, ClusterIP, LoadBalancer, and ExternalName
     - Workload controllers ReplicaSet, ReplicationController, Deployments, StatefulSets, DaemonSet, Job, CronJob
-    - ConfigMap, Secrets: as environment variables or volumes
+    - ConfigMap, Secrets: as environment variables or volumes (Volume subpath does not work)
     - Resource limits
     - Pod & container metrics
     - Horizontal Pod Autoscaling
+    - Volumes can be shared between containers in a Pod
```

Comment:
It would be helpful to group pod-related features together in this list, for readability.

Reply:
Nit: This is still an issue, but not a blocker for merge.

I'd like to see something like:
* Pod
  * single and multiple containers
  * resource limits
  * liveness and readiness probes
  * ConfigMap and Secret environment variables and volumes (except volume subpath does not work)
  * volumes can be shared between containers within a Pod
  * etc.
* Workload controllers
  * ReplicaSet / ReplicationController
  * Deployment
  * Job
  * etc.
* Service types ...
* HorizontalPodAutoscaler, except ...
* etc.